app/components/EventInfo.tsx

```diff
@@ -0,0 +1,98 @@
+import { View, ViewStyle } from "react-native"
+import React from "react"
```

Comment:
Only global components go directly in the /components folder. So maybe add this into components/Dashboard/